Integrations/SecurityAdvisor/SecurityAdvisor_description.md

```diff
@@ -0,0 +1,95 @@
+3. A user is targeted with ransomware attack. Use coach-end-user end user command with this user's email address and "ransomware" context to send them a training on staying safe online.
+4. You can create conditional coaching conditions like send coaching is the user has scored less than 80 in a particular coaching context.
+
+You can add coach-end-user command (see commands below) to any section of your playbook to trigger these notifications.
+
+## Prerequisites
+---
+To get your  __API KEY__ log into www.securityadvisor.io and get your key from "My Profile" section or contact us at support@securityadvisor.io
+
+## Configure SecurityAdvisor on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for SecurityAdvisor.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __use system proxy__
+    * __trust any certificate__
+    * __API Endpoint URL__ = "https://www.securityadvisor.io/
+    * __api key__ = See Prerequisites above to get your API key
+4. Click __Test__ to validate the URLs, token, and connection.
+## Fetched Incidents Data
+---
+
+## Commands
+---
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+1. coach-end-user
+### 1. coach-end-user
+---
+sends contextual message to single user. This command takes a user email address as "user" input. This is where the training email is sent.
+"context" input has four pre-defined settings:
+1. malware: Coach user on malware
+2. phishing: Coach user on phishing
+3. ransomware: Coach user on ransomware
+4. spam: Coach user on avoiding spam.
+
+##### Base Command
+
+`coach-end-user`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| user | user email address | Required | 
+| context | coaching context | Optional | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| SecurityAdvisor.CoachUser.coaching_date | string | When coaching was sent or completed | 
+| SecurityAdvisor.CoachUser.coaching_status | string | User coaching status for context. "Pending" is coaching has been sent and is pending. "Done" is user has completed the coaching. | 
+| SecurityAdvisor.CoachUser.coaching_score | string | User's coaching score out of 100 | 
+| SecurityAdvisor.CoachUser.context | string | Coaching context | 
+
+##### Command Example
+`coach-end-user user="track@securityadvisor.io" context="phishing"`
+
+##### Context Example
+```
+{
+    "SecurityAdvisor.CoachUser": {
+        "coaching_date": "2019-10-04T21:04:19.480425", 
+        "coaching_status": "Pending", 
```

Comment:
so if there is a status, how the user should know if coaching status is completed?

Reply:
Currently the user will logon to securityadvisor.io to see detailed reporting. We will be working on importing data to demisto and building dashboards shortly.